configure/options.md

```diff
@@ -0,0 +1,39 @@
+Changing MySQL Options
+============================================================================
+
+MySQL allows the option to configure the database with a configuration file. You can pass the MySQL configuration options to the cluster in the following ways:
+* CR.yaml
+* ConfigMap
+
+### Edit the CR.yaml
+
+Edit the configuration section of the deploy/cr.yaml. See the [PXC section]( https://percona.github.io/percona-xtradb-cluster-operator/configure/operator).
+
+
+### Use a Configmap
+
+A configmap allows a user to separate configurations from the pods. Configmaps store unencrypted configuration settings. For sensitive information, the user should use Secrets.
```

Comment:
It may be useful to avoid mentioning encryption and encrypted settings here, as this exact doc is about unencrypted options from my.cnf,  and mentioning encryption may drive some readers to the idea that it's possible to keep some ordinary my.cnf options hidden and cyphered :)